plugins/woocommerce/src/Blocks/BlockTypes/ProductGallery.php

```diff
@@ -125,7 +126,7 @@ protected function render( $attributes, $content, $block ) {
 		}
 
 		$number_of_thumbnails           = $block->attributes['thumbnailsNumberOfThumbnails'] ?? 0;
-		$classname                      = $attributes['className'] ?? '';
+		$classname                      = StyleAttributesUtils::get_classes_by_attributes( $attributes );
```

Comment:
For safety reason, I would pass the second parameter: 
```suggestion
		$classname                      = StyleAttributesUtils::get_classes_by_attributes( $attributes, array('extra_classes') );
```

Furthermore, `get_classes_by_attributes` returns an object with `classes` and `styles` keys. We need to pick the `class` property.

Reply:
Actually `get_classes_by_attributes` returns the `classes` property 😊, added the `extra_classes` property to the parameter.